pgmax/fg/graph.py

```diff
@@ -70,6 +70,17 @@ def __post_init__(self):
             0,
             0,
         )
+
+        # Useful objects to build the FactorGraph
+        self._factor_types_to_groups: OrderedDict[
+            Type, List[groups.FactorGroup]
+        ] = collections.OrderedDict(
+            [(factor_type, []) for factor_type in FAC_TO_VAR_UPDATES]
+        )
```

Comment:
Move this back to avoid unnecessary diffs

Reply:
The comment `# See FactorGraphState docstrings for documentation on the following fields` does not apply to this object